integrations/gitlab/scaGitLabScript.py

```diff
@@ -199,19 +206,44 @@ def to_hungarian_case(input_string):
     hungarian_case_words = input_string[0].upper() + input_string[1:].lower()
     return hungarian_case_words
 
+def str_to_bool(value):
+    return value.lower() in ("true", "1", "yes", "on")
```

Comment:
@sebasrevuelta i don't see this function being used anywhere? we probably don't need it since you are already doing `value.lower()` later on anyways

Reply:
You are right, I developed it when we had the variable `lowering-unreachable ` coming from the environment, but with CLI args, we don't need that anymore! Thanks @sinat101